src/Scripts/Ecs/Systems/Skills/BowAttackSystem.gd

```diff
@@ -0,0 +1,21 @@
+extends "res://Scripts/Ecs/Core/System.gd"
+
+var _event_bus
+
+func _init(event_bus):
+	required_component_types = ["HealthComponent"]
+	_event_bus = event_bus
+	_event_bus.connect("bow_attack", self, 'activate')
+
+func activate(target: Vector2):
+	print("bow: " + String(target))
+	var target_entity = _get_entity_at(target)
+	if target_entity != null:
```

Comment:
We eventually should care about line-of-sight. When that happens, whose responsibility will it be, to make sure the target is in the line-of-sight, before we apply damage?

Whoever it is, please add a comment appropriately. I think it makes sense to add it here.